frontend/src/utils/graphql.ts

```diff
@@ -0,0 +1,2 @@
+/** Utility type for getting the type of a generated GraphQL type without the `__typename` property. */
+export type GraphqlType<T extends { __typename?: any }> = Omit<T, "__typename">;
```

Comment:
Fair enough - my thought behind this was that omitting `__typename` somewhat decouples these types from GraphQL, though I think it may indeed just be an unnecessary wrapper. Will remove it.